SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec

```diff
@@ -85,6 +85,8 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %config %{_localstatedir}/lib/initramfs/kernel/%{uname_r}
 
 %changelog
+*   Fri Oct 02 2020 Chris Co <chrco@microsoft.com> 5.4.51-10
+-   Update release number
```

Comment:
Nit: maybe also say why we're updating that release number. Otherwise this status doesn't really say much. I'm guessing that we need to match whatever `kernel.spec` produces.

Reply:
Correct. The `kernel-signed-<arch>.spec` version and release must match the `kernel.spec` version/release. Updated the changelog entries to reflect this.